### PR TITLE
Fix bug introduced by change in Twitter API, update dependencies

### PR DESCRIPTION
**Pull request by alexpkeil1**
- Simple change from "String" to "Array[]" in the Users type for a single parameter
- Updated dependency version ranges, which were really strict and leaving really old versions of some important packages (especially DataFrames) and starting to cause errors when used outside of a really stringent virtual environment. I'd be happy to change how I did the versioning requirements because it may be too wide of a range of versions for your preferences.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 OAuth = "22d8b318-f366-56fb-a292-a93f7d76c017"
 
 [compat]
-DataFrames = "~0.18.0"
-JSON = "~0.21.0"
-OAuth = "~0.7.1"
+DataFrames = ">= 1.1"
```
Comment (randyzwitch) [inline Project.toml:12]: What's the current thinking behind only minimum versions specified? At one time, this wasn't recommended, though I don't have strong objections to it. Whatever the auto-merge rules for the General registry are is fine with me.

File: Manifest.toml [hunk 2]
```diff
@@ -0,0 +1,295 @@
+# This file is machine-generated - editing it directly is not advised
```
Comment (randyzwitch) [inline Manifest.toml:1]: Are manifest files supposed to be committed? Isn't this project-specific?

**Comment by alexpkeil1** [comment]
You're right. I'll update the pull request with this removed and the versioning following the general guidelines, when I can find them

**Comment by alexpkeil1** [comment]
Submitting new pull request

**Pull request closed without merging**